Readme.md

```diff
@@ -1,176 +1,341 @@
+
+**Note: It is highly recommended to [create a new user](#Creatinganewuser) for dev purposes especially if you currently use the extension and would like to develop without interfering in it's daily use**
+
+### Creating a New User
+*Chrome:*
+1. In the top right corner of your browser click the user icon (above the menu). Alternatively go into [settings](chrome://settings/)
+2. Now click **Manage People**, in settings: Under *People* click **Manage other people**
+3. Now click **Add People** in the lower left
+4. Choose a name such as **Worldbrain Test** and any icon. 
+5. All Finished! :tada:
+
+*Firefox:*
+1. In firefox url type in [about:profiles](about:profiles)
+2. Click `Create a new Profile`
+3. Hit `continue`
+4. Now enter a name such as **Worldbrain Test** and hit `continue`
+5. You should see the new Profile and can now click `Launch profile in new browser`
+6. All Finished! :tada: For more info see: [creating multiple profiles](https://developer.mozilla.org/en-US/Firefox/Multiple_profiles)
+
+### Running + Debugging
+*Chrome*
+1. Open a `New Tab`
+2. Click on the `Menu`
+3. Go to `More Tools`
+4. Click `Extensions`
+5. At this point it is reccomended to bookmark `Extensions` for ease of use in development
+6. Click `Load unpacked extension...` 
+7. Now navigate to the folder where you cloned the repo and there should be a new folder named extension (this was created by [`yarn watch`]) go into this folder then click `select this folder`
+8. Everything should be all loaded! 😃
+9. To view developer tools go to the [Extension Page](chrome://extensions/) and under *inspect views:* click `background page`
+
+*Firefox*
+1. Enter [about:debugging](about:debugging) into the url 
+2. Check the `Enable add-on debugging` box
+3. Click `Load Temporary Add-on`
+4. Now navigate to the folder where you cloned the repo and there should be a new folder named extension (this was created by [`yarn watch`]) go into this folder select the `manifest.json` file and then click `open`
+5. Everything should be all loaded! 😃
+6. To view the developer tools simply click `Debug` under the Worldbrain Extension in [about:debugging](about:debugging)
+
+**Now you are ready to hack! 😃** 
+We reccomend reading through the [Code Overview](#CodeOverview) to get an idea of how the extension works 
+
+
+### <a id="code-overview" href="#code-overview">Code Overview</a>
+
+### <a id="a-brief-overview-of-web-e" href="#a-brief-overview-of-web-e">A brief overview of Web Extensions</a>
+
+A web extension consists of three main parts:
+
+- `background.js` always runs, in an 'empty invisible tab', listening for
+  messages and events.
+- `content_script.js` is loaded into every web page that is visited. It is
+  invisible from that web page's own scripts, and can talk to the background
+  script.
+- `overview/overview.html`, with the resources in that folder, provides the main

```

Comment:
this one is more specific to our extension only, rather than web extensions in general. It is just the Overview UI's HTML file. So maybe could either remove this line, replace it with a general note about UI scripts/pages, or other lines for both `options.html` and `popup.html`, which both function in the exact same way that `overview.html` does?